packaging/debian/control

```diff
@@ -0,0 +1,38 @@
+Source: ansible-runner
+Section: admin
+Priority: optional
+Maintainer: Ansible <info@ansible.com>
+Build-Depends: debhelper (>= 11), dh-python, python-all, python-setuptools, python3-all, python3-setuptools, python-pytest, python3-pytest, python-pexpect (>= 4.5), python3-pexpect (>= 4.5), python-mock, python3-mock, python-psutil, python3-psutil, python3-yaml
+Standards-Version: 4.1.3
+Homepage: https://www.ansible.com
+X-Python-Version: >= 2.7
+X-Python3-Version: >= 3.5
+Vcs-Git: https://github.com/ansible/ansible-runner.git
+
+Package: python-ansible-runner
+Architecture: all
+Depends: ${python:Depends}, ${misc:Depends}
+Recommends: ansible (>= 2.1)
```

Comment:
Will Ansible versions back to 2.1 actually work with ansible-runner?  Maybe we should be looking for a newer Ansible version.

Reply:
I've only personally tested this with 2.5+ would be interested to see how this works going that far back.

Reply:
My "hello world"-style debug message example worked on Ansible 2.2.1.0, but failed on Ubuntu 16.04's Ansible version (2.0.0.2, if I remember correctly).  Runner is making use of a setting that was only made available starting in 2.1.

Reply:
Do we have a set of versions that runner explicitly supports?

Reply:
I don't think we have an explicit list.  Leaving this at >= 2.1 is probably fine.